src/equations/passive_tracers.jl

```diff
@@ -6,14 +6,14 @@
 #! format: noindent
 
 @doc raw"""
-    PassiveTracerEquations(flow_equations)
+    PassiveTracerEquations(flow_equations; n_tracers)
 
 Adds passive tracers to the `flow_equations`. The tracers are advected by the flow velocity.
 These work for arbitrary NDIMS with arbitrary numbers of tracers NTracers. For one dimension, with 
```

Comment:
```suggestion
These work for arbitrary dimensions with arbitrary numbers of tracers `n_tracers`. For one dimension, with 
```
`NDIMS` and `NTracers` are not defined in the docstring.